elastic/kibana
# [Cloud Posture] Create concrete csp rules post create package

create concrete rules per CSP package policy using PostCreate callback.
working with the following Integration branch
- https://github.com/elastic/integrations/pull/3081

To test it: 
1. add our package using the relevant integration branch -> check for CSP rules in the saved objects page
2. Delete the integration -> make sure the rules were removed.
